libs/core/src/components/sage-divider/sage-divider.tsx

```diff
@@ -0,0 +1,44 @@
+import { Component, Prop, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'sage-divider',
+  styleUrl: 'sage-divider.scss',
+  shadow: true,
+})
+export class SageDivider {
+  /**
+   * Sets divider to display vertically
+   * @defaultValue false
+   */
+  @Prop() vertical = false;
```

Comment:
Does it make sense to have this prop be named `direction` or `variant` and pass a default value of `horizontal`?

Reply:
I don't understand the benefit or difference here. Also, seems the logic is much simpler when using use a boolean and just one class, but maybe I'm missing something.

Reply:
Yep, could be fine either way. In my mind, having a divider be horizontal by default feels like it makes the vertical a secondary option where as defining `direction` makes each equal.

Definitely not a thing that you need to change if you don't like that solution.

Reply:
Another topic we can discuss further in our upcoming meeting, but I think the intention here was to better align with our definition of a `variant` for consistency.